File: Tyche.iOS/Pool/Sources/Pool/Domain/PoolGamblerScore.swift
public struct PoolGamblerScore: Codable {
    let poolId: String
    let poolName: String
    let gamblerId: String
    let gamblerUsername: String
    let currentPosition: Int?
    let beforePosition: Int?
    let score: Int?
}

extension PoolGamblerScore {
    func toPoolGamblerScoreModel() -> PoolGamblerScoreModel {
        return PoolGamblerScoreModel(
            poolId: self.poolId,
            poolName: self.poolName,
            gamblerId: self.gamblerId,
            gamblerUsername: self.gamblerUsername,
            currentPosition: self.currentPosition,
            beforePosition: self.beforePosition,
            score: self.score
        )
    }
}

File: Tyche.Android/data/pool/src/main/java/com/felipearpa/tyche/data/pool/domain/PoolGamblerScore.kt
package com.felipearpa.tyche.data.pool.domain

data class PoolGamblerScore(
    val poolId: String,
    val poolName: String,
    val gamblerId: String,
    val gamblerUsername: String,
    val currentPosition: Int?,
    val beforePosition: Int?,
    val score: Int?
)
